release-content/0.15/release-notes/13454_vbao.md

```diff
@@ -0,0 +1,13 @@
+**Bevy 0.15** introduces a new Screen Space Ambient Occlusion (SSAO) algorithm: [Visibility Bitmask Ambient Occlusion](https://cybereality.com/screen-space-indirect-lighting-with-visibility-bitmask-improvement-to-gtao-ssao-real-time-ambient-occlusion-algorithm-glsl-shader-implementation/) (VBAO). VBAO builds on GTAO by adding a bitmask that allows multiple "sectors" of a considered half circle to be occluded, instead of just one angle. This improves the accuracy of the technique:
```

Comment:
I think we should've linked to the actual paper https://arxiv.org/abs/2301.11376 here instead

Reply:
```suggestion
**Bevy 0.15** introduces a new Screen Space Ambient Occlusion (SSAO) algorithm: [Visibility Bitmask Ambient Occlusion](https://arxiv.org/abs/2301.11376) (VBAO). VBAO builds on GTAO by adding a bitmask that allows multiple "sectors" of a considered half circle to be occluded, instead of just one angle. This improves the accuracy of the technique:
```